app/models/nic/interface.rb

```diff
@@ -74,9 +74,11 @@ def identifier_change
       old_value = self.identifier_was || ''
```

Comment:
Note there is a comment on line 71 stating that a change to/from virtual/physical is disallowed, but this should be updated to state that you're now permitting a change from physical to virtual (but you're not allowing virtual to physical, for whatever reason.)